lib/xpm_ruby.rb

```diff
@@ -2,6 +2,7 @@ module XpmRuby
   class Error < StandardError; end
   class Unauthorized < Error; end
   class AccessTokenExpired < Unauthorized; end
+  class Forbidden < Unauthorized; end
```

Comment:
hmm... we sure about that one? 403 doesn't really inherit from 401?

Was thinking `class Forbidden < Error; end` as an authentication error is different from authorization one.